packages/mui-material/src/Alert/Alert.js

```diff
@@ -59,7 +60,7 @@ const AlertRoot = styled(Paper, {
       padding: '6px 16px',
       variants: [
         ...Object.entries(theme.palette)
-          .filter(([, value]) => value && value.main && value.light)
+          .filter(([, value]) => value && checkSimplePaletteColorValues(value, ['light']))
```

Comment:
I think it can be a little bit easier to use with:
```suggestion
          .filter(checkValidPalette(['light']))
```

Reply:
Updated 👍🏼